berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java

```diff
@@ -0,0 +1,133 @@
+                RegularStatement toPrepare = new SimpleStatement(prepStatement.getQuery());
+                com.datastax.driver.core.PreparedStatement prepared = session.prepare(toPrepare);
+                preparedStatements.put(prepStatement.getId(), prepared);
+            }
+        }
+        this.async = async;
+    }
+
+    /**
+     * Accepts two possible combinations:
+     * <ul>
+     * <li><b>Query</b>, which contains:
+     * <ul>
+     * <li>consistencyLevel - Consistency level of statement which will be executed.</li>
+     * <li>query - String representation of CQL statement which will be executed.</li>
+     * </ul>
+     * </li>
+     * <li><b>Prepared statement</b>, which contains:
+     * <ul>
+     * <li>consistencyLevel - Consistency level of statement which will be executed.</li>
+     * <li>preparedStatementId - Id of prepared statement to execute (defined within constructor).</li>
+     * <li>values - List of values to bind to prepared statement's variables.</li>
+     * </ul>
+     * </li>
+     * </ul>
+     * Depending on the map content, appropriate option will be executed (either query or prepared statement).
+     */
+    @SuppressWarnings("unchecked")
+    @Override
+    public void accept(Map<String, Object> queryMetadata) {
+        ConsistencyLevel consistencyLevel = getConsistencyLevel(queryMetadata);
+        String statement = (String) queryMetadata.get(QUERY);
+        Statement toExecute = null;
+        if (statement != null) {
+            toExecute = new SimpleStatement(statement);
+        } else {
+            String statementId = (String) queryMetadata.get(PREPARED_STATEMENT_ID);
+            if (statementId == null || statementId.isEmpty()) {
+                throw new InvalidQueryMetadataException("Statement id cannot be null nor empty");
+            }
+            List<Object> values = (List<Object>) queryMetadata.get(VALUES);
+            if (values == null || values.isEmpty()) {
+                throw new InvalidQueryMetadataException("Values cannot be null nor empty");
+            }
+            com.datastax.driver.core.PreparedStatement preparedStatement = preparedStatements.get(statementId);
+            if (preparedStatement == null) {
+                throw new InvalidQueryMetadataException(
+                        "Prepared statement with id: " + statementId + " not defined within configuration.");
+            }
+            toExecute = preparedStatement.bind(values.toArray(new Object[values.size()]));
+        }
+
+        toExecute.setConsistencyLevel(consistencyLevel);
+        if (async) {
+            session.executeAsync(toExecute);
+        } else {
+            session.execute(toExecute);
+        }
+    }
+
+    private ConsistencyLevel getConsistencyLevel(Map<String, Object> queryMetadata) {
+        String consistency = (String) queryMetadata.get(CONSISTENCY_LEVEL);
+        if (consistency == null || consistency.isEmpty()) {
```

Comment:
Idea was to have it per request, not global. Having it optional with fallback to ONE is still OK, and probably a bit more user friendly. I will make it optional.